ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -0,0 +1,198 @@
+import NetRegexes from '../../../../../resources/netregexes';
+// import Outputs from '../../../../../resources/outputs';
+// import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  bodyActor?: PluginCombatantState;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
+  triggers: [
+    {
+      id: 'P4N Elegant Evisceration',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A50', source: 'Hesperos' }),
+      response: Responses.tankCleave(),
+    },
+    // Strong proximity Aoe
+    {
+      id: 'P4N Levinstrike Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3F', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to Corners',
+        },
+      },
+    },
+    // Knockback from middle
+    {
+      id: 'P4N Well Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3E', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
```

Comment:
Maybe "Middle Knockback" or "Knockback from Middle"? I think "knockback" on its own is too vague for this fight.